src/compile/facet.ts

```diff
@@ -242,6 +233,13 @@ export class FacetModel extends ModelWithField {
     );
   }
 
+  private columnDistinctSignal() {
+    // In facetNode.assemble(), the name is always this.getName('column') + '-layout'.
+    const facetLayoutDataName = this.getName('column') + '-layout';
```

Comment:
Will need to fix this if you rename the data source. I'm also happy to use - everywhere instead.